data/human/human missions.txt

```diff
@@ -6413,12 +6413,12 @@ mission "Moving House 1"
 				
 			label sure
 			`	They stop in surprise, then nod briefly. "I'll be at your ship within an hour.`
-			`	An hour later, you see them approach the <ship> carrying a small suitcase. They've exchanged their formal appearance for a button-down shirt and skirt with their hair loose, hanging down to their shoulders. They walk right up to you and stick out a hand. "Kris Brecht."`
+			`	An hour later, you see them approach the <ship> carrying a small suitcase. They've exchanged their formal appearance for a button-down shirt and skirt with their hair loose, hanging down to their shoulders. They walk right up to you and stick out a hand. "Kris Brecht. I use they/them."`
 			choice
 				`	(Shake their hand.)`
 					goto shake
 				`	(Just stare.)`
-			`	You stare at Kris' unorthodox getup while they stand there with their hand out for a few beats. Then they sigh and take it back. "Will this be a problem?" They gesture to themselves.`
+			`	You pause at the mention of Kris' pronouns while they stand there with their hand out for a few beats. Then they sigh and take it back. "Will this be a problem?" They gesture to themselves, indicating both the pronouns and the very unorthodox outfit.`
```

Comment:
Not sure if this is needed, just showing the pronouns feels like enough. Staring at an "unorthodox getup" is reasonable, but not at someone's pronouns.